modules/update-using-custom-machine-config-pools-about.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * updating/update-using-custom-machine-config-pools.adoc
+
+[id="update-using-custom-machine-config-pools-about_{context}"]
+= About performing a canary rollout update
+
+This topic describes the general workflow of this canary rollout update process. The steps to perform each teak in the workflow are decribed in the follwoing sections.
+
+. Create MCPs based on the worker pool. The number of nodes in each MCP depends on a few factors, such as your maintaince window duration for each MCP, and the amount of reserve capacity (extra worker nodes) available in your cluster.
++
+[NOTE]
+====
+You can change the `maxUnavailable` setting in a MCP to specify the percentage or the number of machines that can be updating at any given time. The default is 1.
+====
+
+. Add a labels to the custom MCPs. For each node that you do not want to update simultaneously with the rest of the cluster, add a `<key>=<value>` label to the custom MCP and the nodes. This label associates the node to the MCP.
++
+[NOTE]
+====
+Do not remove the default worker label from the nodes. The nodes *must* have a role label to function properly in the cluster.
+====
+
+. Pause the MCPs you do not want to update as part of the update process.
++
+[NOTE]
+====
+Pausing the MCP will also pause the kube-apiserver-to-kubelet-signer automatic CA certificates rotation. New CA certificates are generation and removal happens at 292 day (from the installation day) and 365 days respectively. See the link:https://access.redhat.com/articles/5651701[article] to findout how much time you have before the next automatic CA certificate rotation. Make sure the pools are unpaused when the CA cert rotation happens. If the MCPs are paused, the cert rotation does not happen. and that would make the cluster degraded and causes failure in multiple `oc` commands, including but not limited to `oc debug`, `oc logs`, `oc exec`, and `oc attach`.
```

Comment:
@xingxingxia  Could someone from your team to help review CA cert rotate description here? Thanks!

Reply:
@wangke19 , could you help take a look? Thank you.

Reply:
` happens at 292 day` I think it is ` happens at 292th day` . Other than that, lgtm.